payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardDetailsSectionController.kt

```diff
@@ -26,4 +26,23 @@ class CardDetailsSectionController(
     internal val isStripeCardScanAvailable = DefaultIsStripeCardScanAvailable()
 
     override val error = cardDetailsElement.controller.error
+
+    // The following functions are only for testing purposes.
```

Comment:
Should we make them extension functions in the test classes instead then?

Reply:
The problem is that the sub elements and controllers are internal and not in the same package as the tests where I'm trying to use them, so I needed to expose a function to access them, I assume it would be the same case if I try to create the extension functions outside of this package.
I also tried to change the visibility of the dependencies to make them `VisibleForTesting`, but there are a lot of them (like A LOT), so it would've been a much larger change, so I decided to do this instead.